src/main/java/games/strategy/triplea/attachments/RulesAttachment.java

```diff
@@ -222,11 +221,11 @@ public void setBattle(final String value) throws GameParseException {
   }
 
   @GameProperty(xmlProperty = true, gameProperty = true, adds = false)
-  public void setBattle(final ArrayList<Tuple<String, ArrayList<Territory>>> value) {
+  public void setBattle(final List<Tuple<String, List<Territory>>> value) {
```

Comment:
It looks like you were purposefully avoiding changing types on attachment mutators.  Is there a reason why this one is different?

**EDIT:** After further review, there's a few more mutators that have had their parameter types changed.  So my question should be more general and becomes how did you choose which mutators to modify and which to leave as-is?

Reply:
Because those changes were done by hand:
The ones I noticed were changed the others stayed ^^
Will fix that